packages/esm-appointments-app/src/workload/monthly-view-workload/monthly-view.component.tsx

```diff
@@ -0,0 +1,54 @@
+import dayjs from 'dayjs';
+import React, { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import DaysOfWeekCard from '../../calendar/monthly/days-of-week.component';
+import { monthDays } from '../../helpers';
+import SelectedDateContext from '../../hooks/selectedDateContext';
+import styles from './monthly-workload.scss';
+import MonthlyWorkloadCard from './monthlyWorkCard';
+
+interface MonthlyCalendarViewProps {
+  calendarWorkload: Array<{ count: number; date: string }>;
+  dateToDisplay?: string;
+}
+const daysInWeek = ['SUN', 'MON', 'TUE', 'WED', 'THUR', 'FRI', 'SAT'];
```

Comment:
Can we wrap this in translate fields so they can be localized?

Reply:
> Can we wrap this in translate fields so they can be localized?

Thank you. It's fixed

Reply:
@gabriel090 where is the fix? I can't see it